src/lib/expert_partitioner/tree_views/disks.rb

```diff
@@ -108,70 +97,57 @@ def items
       end
 
       return ret
-
     end
 
-
     def do_create_partition
-
       begin
-        @disk.partition_table()
+        @disk.partition_table
       rescue Storage::WrongNumberOfChildren, Storage::DeviceHasWrongType
         Yast::Popup::Error("Disk has no partition table.")
         return
       end
 
-      CreatePartitionDialog.new(@disk).run()
+      CreatePartitionDialog.new(@disk).run
 
       update(true)
-
     end
 
-
     def do_format
-
       sid = Yast::UI.QueryWidget(Id(:table), :CurrentItem)
 
       storage = Yast::Storage::StorageManager.instance
-      staging = storage.staging()
+      staging = storage.staging
       device = staging.find_device(sid)
 
       begin
-        blk_device = Storage::to_blk_device(device)
+        blk_device = Storage.to_blk_device(device)
         log.info "do_format #{sid} #{blk_device.name}"
-        FormatDialog.new(blk_device).run()
+        FormatDialog.new(blk_device).run
       rescue Storage::DeviceHasWrongType
         log.error "do_format on non blk device"
       end
 
       update(true)
-
     end
 
-
     def do_delete_partition
-
       sid = Yast::UI.QueryWidget(Id(:table), :CurrentItem)
 
       storage = Yast::Storage::StorageManager.instance
-      staging = storage.staging()
+      staging = storage.staging
 
       device = staging.find_device(sid)
 
       begin
-        partition = Storage::to_partition(device)
+        partition = Storage.to_partition(device)
       rescue Storage::WrongNumberOfChildren, Storage::DeviceHasWrongType
         Yast::Popup::Error("Only partitions can be deleted.")
         return
       end
 
-      if RemoveDescendantsPopup.new(partition).run()
-        staging.remove_device(partition)
-        update(true)
-      end
-
+      return unless RemoveDescendantsPopup.new(partition).run
```

Comment:
just my matter of taste but I prefer for better readability empty line between early exit and rest of method to more emphasise it.